nautobot/nautobot_data_validation_engine/models.py

```diff
@@ -42,13 +38,19 @@ def get_for_model(self, content_type):
 class ValidationRule(PrimaryModel):
     """Base model for all validation engine rule models."""
 
-    name = models.CharField(max_length=100, unique=True)
+    name = models.CharField(max_length=CHARFIELD_MAX_LENGTH, unique=True)
+    field = models.CharField(
+        max_length=50,
```

Comment:
Any reason not to make this use CHARFIELD_MAX_LENGTH as well?

Reply:
It is from this [comment](https://github.com/nautobot/nautobot/pull/6866#discussion_r1944974196) right here. We definitely could change it to CHARFIELD_MAX_LENGTH if you think this is necessary.

Reply:
I think it would be a good safety to make the change yeah.